apps/sms/views/conversation/js/conversation.js

```diff
@@ -1910,9 +1910,15 @@ var ConversationView = {
     }
     params.items.push(subjectItem);
 
-    // If we are on a thread, we can call to SelectMessages
+    /*
+     * If we are on a thread,
+     * we can call Delete Thread, SelectMessages or Message Settings
+     */
```

Comment:
__nit:__ use just ```//``` comment style and use all available space (up to 80 chars in a line)